source/release-notes.txt

```diff
@@ -10,6 +10,37 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Enable "Modern Dark Mode" by default
+- Surface type warnings (:issue:`COMPASS-6543`)
+- Combine array fields into one in the import CSV preview (:issue:`COMPASS-6766`)
+- Remove export feature flag, use new export, remove old code (:issue:`COMPASS-6582`)
+- Add settings to sidebar menus (:issue:`COMPASS-6796`)
+- Show password only when user focuses on input (:issue:`COMPASS-6161`)
+- Add autocomplete support for $percentile, $median ans $$USER_ROLES (:issue:`COMPASS-6780`) (:issue:`COMPASS-6781`)
+- Toggle state of "Edit connection string" in connection form is now judged also based on the new global preference
```

Comment:
I'm not sure I understand what this bullet means. It sounds like something that happens behind the scenes, so maybe it can be removed?

Reply:
I checked this is still relevant to outside users so will leave it in.

Reply:
@ianf-mongodb which ticket is still relevant to external users? I agree that the QE test data sounds pretty internal.

Reply:
It means that there's a setting that users can change inside the Compass UI that controls the default behavior of the "Edit connection string" toggle in the connection form, which is relevant because that toggle controls whether the password is visible when a new connection is created.

Reply:
Ok I added a bit more detail to this bullet:

```
- Toggle state of **Edit connection string** is based on the new global preference. This setting controls whether a password is visible when creating a new connection.
 ```